src/AnimalSoundsApp.java

```diff
@@ -1,2 +1,56 @@
+// Definim interfața Animal
```

Comment:
comment-urile si limba de programare sunt engleza; la fel cum nu ne-ar placea sa traducem comment-uri din germana, rusa sau indiana

Reply:
pun comment-urile la pull request in romana pentru ca suntem intr-o "echipa" romaneasca la o facultate romaneasca; si pull request ul se inchide; dar commenturile fac parte din codul sursa;